modules/ucfunnelBidAdapter.js

```diff
@@ -71,6 +71,9 @@ export const spec = {
       ttl: 1800
     };
 
+    if (bidRequest.params && bidRequest.params.bidfloor && ad.cpm && ad.cpm < bidRequest.params.bidfloor) {
+      bid.cpm = 0;
```

Comment:
Why would  you set the bid to zero instead of the floor? Is it because the bid must be invalid so you're throwing it out?

Reply:
the bid is invalid, because the price is under the floor price. I set the bid to zero to throw it out.